cmd/sovereignnode/config/enableEpochs.toml

```diff
@@ -158,7 +158,7 @@
     FrontRunningProtectionEnableEpoch = 1
 
     # IsPayableBySCEnableEpoch represents the epoch when a new flag isPayable by SC is enabled
-    IsPayableBySCEnableEpoch = 1
+    IsPayableBySCEnableEpoch = 0
```

Comment:
This is not enough.

We need more changes in this PR:
- cmd/node/config/config.toml (we will discuss how to address this changes)
- Add these changes:
```
  updateTOMLValue enableEpochs.toml "SCDeployEnableEpoch" 0
  updateTOMLValue enableEpochs.toml "BuiltInFunctionsEnableEpoch" 0
  updateTOMLValue enableEpochs.toml "ESDTEnableEpoch" 0
```
- Update go mods for es indexer from this PR: https://github.com/multiversx/mx-chain-go/pull/5635